pkg/ucp/integrationtests/planes/aws_test.go

```diff
@@ -35,15 +34,15 @@ const (
 )
 
 func Test_AWSPlane_PUT_Create(t *testing.T) {
-	server := testserver.StartWithETCD(t, api.DefaultModules)
+	server := testhost.Start(t)
 	defer server.Close()
 
 	response := server.MakeFixtureRequest("PUT", awsPlaneResourceURL, awsPlaneRequestFixture)
 	response.EqualsFixture(200, awsPlaneResponseFixture)
 }
 
 func Test_AWSPlane_PUT_Update(t *testing.T) {
-	server := testserver.StartWithETCD(t, api.DefaultModules)
+	server := testhost.Start(t)
```

Comment:
This is not only simpler, but it uses the in-memory database so it's a **lot** faster to run.